lib/crowbar/init/application.rb

```diff
@@ -109,12 +109,12 @@ class Application < Sinatra::Base
             body: nil
           )
         else
-          logger.error(migration[:stdout_and_stderr])
+          logger.error(migration[:message])
```

Comment:
good catch! but there is another occurence in the `helpers.rb` in `crowbar_init`. could you please rather change the `message` to `stdout_and_stderr` in `run_cmd` ?